cookbook/profiler/profiling_data.rst

```diff
@@ -0,0 +1,61 @@
+.. index::
+   single: Profiling; Profiling data
+
+How to Access Profiling Data Programmatically
+=============================================
+
+Most of the times, the profiler information is accessed and analyzed using its
+web-based visualizer. However, you can also retrieve profiling information
+programmatically thanks to the methods provided by the ``profiler`` service.
+
+When the response object is available, use the
+:method:`Symfony\\Component\\HttpKernel\\Profiler\\Profiler::loadProfileFromResponse`
+method to access to its associated profile::
+
+    // ... $profiler is the 'profiler' service
+    $profile = $profiler->loadProfileFromResponse($response);
+
+When the profiler stores data about a request, it also associates a token with it;
+this token is available in the ``X-Debug-Token`` HTTP header of the response.
+Using this token, you can access the profile of any past response thanks to the
```

Comment:
i am not sure if this is a very native expression, however i am seeing several `thanks to` in the text. I know @javiereguiluz is spanish :+1: so maybe that is getting through here a bit. But if people is ok with that I am too :older_man: